src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -576,6 +576,11 @@ class Testcase(Model):
   # Note that the number is specific to the repository.
   github_issue_num = ndb.IntegerProperty()
 
+  # Whether the testcase was uploaded.
+  # True by default since this will determine what is put in the regression
+  # corpus.
+  uploaded = ndb.BooleanProperty(default=True)
```

Comment:
Can we rename this to "untrusted"? 

I don't think we're inherently interested in whether this was uploaded by somebody or not. Rather, it's the fact that it's untrusted.